### PR TITLE
improved ignore rules

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: .bowerignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/.idea/
```
Comment (i-like-robots) [inline .bowerignore]: should this be covered by people's global gitignore?

File: secret-squirrel.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
 	files: {
-		allow: [],
+		allow: ['.bowerignore'],
```
Comment (i-like-robots) [inline secret-squirrel.js]: npmignore too?

Comment (wheresrhys) [inline secret-squirrel.js]: already allowed in the global rules 🤷‍♂️

File: .bowerignore (hunk 1 above)
Comment (wheresrhys) [inline .bowerignore]: yep

**Pull request merged**